src/css/custom.css

```diff
@@ -194,17 +275,20 @@ article a:focus {
 	text-decoration: none;
 }
 
-/* ian: Fix for dark mode later
-.pagination-nav__link:hover .pagination-nav__sublabel {
-  color: #2e54ea;
-}
-*/
-
 /* Left drawer menu */
+@media (min-width: 997px) {
+	.sidebarViewport_node_modules--pnpm-\@docusaurus\+theme-classic\@2-4-3_react-dom\@17-0-2_react\@17-0-2_typescript\@5-2-2-node_modules-\@docusaurus-theme-classic-lib-theme-DocPage-Layout-Sidebar-styles-module {
```

Comment:
Ahhh of course. These cursed selectors will not work in a production environment. This could be tricky, after all. We may not have access to these elements.